packages/core/src/connectors/aliyun-dm/config-template.md

```diff
@@ -1,3 +1,23 @@
 {
-  // placeholder
+  "accessKeyId": "<your-access-key-id>",
+  "accessKeySecret": "<your-access-key-secret>",
+  "accountName": "no-reply@example.com",
+  "fromAlias": "ExampleSite",
```

Comment:
Better wrap anything the user needs to replace in a pair of angle brackets

Reply:
fixed.